gamemodes/terrortown/gamemode/server/sv_damage_indicator.lua

```diff
@@ -0,0 +1,10 @@
+
+util.AddNetworkString("ttt2_damage_received")
+
+hook.Add("EntityTakeDamage","TTT2DamageIndicator",function(ent,dmg)
```

Comment:
Maybe add this to the GM function, but at the very end. Maybe even `GM:PlayerTakeDamage`. This makes sure that addons that use the hook and modify the damage are handled correctly.